src/app/tests/suites/certification/Test_TC_DISHALM_1_1.yaml

```diff
@@ -11,90 +11,109 @@
-          The list include entries based on feature support:
-          - 0x0001: SHALL be included if and only if DISHALM.S.F00(Reset).
-          The list SHALL NOT contain any additional values in the standard or scoped range: (0x0000_0000 - 0x0000_4FFF) and (0x0000_F000 - 0x0000_FFFE)
-
-          The list MAY contain values in the Manufacturer Extensible Identifier (MEI) range: (0xXXXX_0000 - 0xXXXX_4FFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1), these values SHALL be ignored.
-
-          The list SHALL NOT contain any values in the Test Vendor or invalid range: (0x0000_5000 - 0x0000_EFFF and 0x0000_FFFF), (0xXXXX_5000 - 0xXXXX_FFFF) and (0xFFF1_0000 - 0xFFFF_FFFF), where XXXX is the allowed MEI range (0x0001 - 0xFFF1)
-      disabled: true
-
-    - label: "Step 5: TH reads from the DUT the EventList attribute."
-      verification: |
-          Verify that the DUT response contains the EventList attribute and have the list of supported events:
-          The list SHALL include all the mandatory entries:
-          0x00
-          This cluster has no entries in the standard or scoped range.
-
-          The list SHALL NOT contain any additional values in the standard or scoped range: (0x0000_0000 - 0x0000_00FF)
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision attribute from the DUT"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS: " !DISHALM.S.F00 "
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label: "TH reads the FeatureMap attribute from the DUT"
+      PICS: "DISHALM.S.F00"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "TH reads the AttributeList attribute from the DUT"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 2, 3, 65528, 65529, 65530, 65531, 65532, 65533]
+
+    - label: "TH reads the attribute(Latch) in AttributeList from the DUT"
+      PICS: DISHALM.S.F00
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [1]
+
+    - label: "Read the global attribute: EventList"
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          value: [0]
+          constraints:
+              type: list
```

Comment:
Again, this is missing the "SHALL NOT" bits.  Followup, comment with link.

Reply:
Reference: https://github.com/project-chip/connectedhomeip/issues/28287